Taskfile.yml

```
@@ -191,6 +191,7 @@ tasks:
       - lint-go-consistent-integration
     cmds:
       - bin/golangci-lint run --config=.golangci-new.yml --new-from-rev=main
+      - go vet -vettool=$(which ./bin/check-switch) ./...
```

```suggestion
      - go vet -vettool=./bin/check-switch ./...
```

There should not be a need to use `which` if we provide a path